packages/f/filc/xmake.lua

```diff
@@ -0,0 +1,26 @@
+package("filc")
+    set_kind("toolchain")
+    set_homepage("https://fil-c.org/")
+    set_description("A memory safe implementation of the C and C++ programming languages.")
+    set_license("LLVM")
+
+    if is_host("linux") then
+        if os.arch() == "x86_64" then
+            set_urls("https://github.com/pizlonator/fil-c/releases/download/v$(version)/filc-$(version)-linux-x86_64.tar.xz")
+            add_versions("0.674", "a8ec349f383a49dacc09a9540643164c67081245e591c1e6823609653f2c9740")
+        end
+    end
+
+    add_deps("patchelf") -- needed for setup.sh
+
+    on_install("@linux|x86_64", function (package)
+        local installdir = package:installdir()
+        os.cp(path.join(os.curdir(), "*"), installdir)
+        os.runv(path.join(installdir, "setup.sh"), {}, {curdir = installdir})
+        package:addenv("PATH", "build/bin")
```

Comment:
installdir/build/bin?

Reply:
pizfix? if it is unused, we should remove it. It will take up additional disk space. and only copy filc to `installdir/bin`, `installdir/include`, `installdir/lib`

if it is used, 

and we should not run `setup.sh`, it will do some system ops.

```sh
ln -s /usr/include/linux .
if test -d /usr/include/x86_64-linux-gnu/asm
then
    ln -s /usr/include/x86_64-linux-gnu/asm .
else
    ln -s /usr/include/asm .
fi
ln -s /usr/include/asm-generic .
cd ../..
set +x
```